src/services/protobuf/assertion.ts

```diff
@@ -23,6 +27,12 @@ export interface IAssertActionStrict extends IAssertAction {
   };
 }
 
+export interface ITransferAssertionActionStrict
+  extends ITransferAssertionAction {
+  assertion_id: string;
```

Comment:
This is a bit of a personal preference, but I think something like this is a bit more explicit about where the `assertion_id` is coming from:

```suggestion
  assertion_id: NonNullable<ITransferAssertionAction['assertion_id']>;
```

And the same thing with the line below.

Reply:
Also this is outside of the changes you made, but I feel like the typings on line 42 could be clearer as well:

```ts
export function getAssertionStateAddress(assertion_id: AssertActionStrict['assertion_id'])
```